django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
@lgh2 , not currently in the code base, but locally I'm using https://github.com/pyvideo/data/pull/293 to detect this, along with verbose tests `make VERBOSE=1 test`